www/modules/custom/upd_search/upd_search.module

```diff
@@ -37,14 +39,15 @@ function upd_search_preprocess_views_view(&$variables) {
   if (!in_array($view->id(), ['search', 'case_studies_index'])) {
     return;
   }
-  $excerpt = [];
-  foreach ($view->result as $result) {
-    $excerpt[$result->search_api_id] = !empty($result->search_api_excerpt) ? $result->search_api_excerpt : '';
-  }
+//  $excerpt = [];
```

Comment:
There you go. Short version: left in case we want to re-use excerpt functionality.